spec.js

```diff
@@ -73,3 +73,53 @@ describe('Utility function dna.util.printf()', () => {
     });
 
   });
+
+////////////////////////////////////////////////////////////////////////////////////////////////////
+describe('Utility function dna.util.realTruth()', () => {
+
+  it('returns a boolean', () => {
+    var dataSet = [
+         { input: '7', expected: true },
+         { input: '0', expected: false }
+         ];
+      function evalData(data) {
+         assert.equal(dna.util.realTruth(data.input), data.expected);
+         }
+      dataSet.forEach(evalData);
+      });
+
+  });
+
+////////////////////////////////////////////////////////////////////////////////////////////////////
+describe('Utility function dna.util.toKebab()', () => {
+
+  it('converts kebab (dashes) name to camelCase', () => {
+    var dataSet = [
+         { input: 'readySetGo', expected: 'ready-set-go' },
+         { input: 'dna',          expected: 'dna' }
+         ];
+      function evalData(data) {
+         assert.equal(dna.util.toKebab(data.input), data.expected);
+         }
+      dataSet.forEach(evalData);
+      });
+
+  });
+
+
+////////////////////////////////////////////////////////////////////////////////////////////////////
+/*
+describe('Utility function dna.util.apply()', () => {
+
+  it('calls fn (string name or actual function) passing in params', () => {
+
```

Comment:
Ah, this is actually a very interesting error.  In a web browser, top-level (global) variables hang off of the `window` object.  Therefore, `app` is available using `window.app`.  However, `window` does not exist by default in Node.js, and the Mocha tests run on Node.js.

The solution is to **register** top-level variables with dna.js so they can be found.  After defining `app`, call:
```javascript
dna.registerContext('app', app);
```
Documentation:
http://dnajs.org/docs/#api-register-context

Example:
https://github.com/dnajs/dnajs-webpack-to-do-app/blob/master/src/js/app.js#L30